utils/const.go

```diff
@@ -0,0 +1,3 @@
+package utils
+
+const UserID = "id"
```

Comment:
best practice for these kind of const vars is to name them exactly alike 
`UserID ="user_id"
or
`ID="id"`
but try to avoid only writing `ID` because in that case, it'll not be clear as to which ID its referring to